apps/aecore/test/multiple_transactions_test.exs

```diff
@@ -279,172 +301,183 @@ defmodule MultipleTransactionsTest do
-          32, 56, 255, 130, 166, 97, 131, 94, 156, 186, 57, 55, 189, 228>>
-      }
+    <<4, 176, 20, 135, 174, 148, 149, 10, 132, 176, 41, 79, 141, 161, 151, 96, 65,
+    70, 198, 93, 25, 11, 90, 105, 57, 41, 39, 255, 197, 140, 163, 9, 180, 126,
+    111, 71, 178, 86, 250, 177, 170, 211, 70, 146, 111, 201, 137, 230, 98, 8,
+    205, 109, 234, 51, 50, 140, 9, 177, 130, 222, 196, 54, 98, 55, 243>>,
+    <<3, 213, 65, 255, 170, 53, 52, 113, 72, 39, 215, 55, 3, 120, 107, 138, 229, 5,
+    32, 56, 255, 130, 166, 97, 131, 94, 156, 186, 57, 55, 189, 228>>
+    }
     account3 = {
-        <<4, 163, 213, 138, 149, 50, 37, 22, 21, 221, 239, 158, 126, 245, 61, 146, 7,
-          15, 86, 26, 224, 169, 46, 191, 199, 39, 172, 189, 146, 10, 111, 160, 51, 7,
-          33, 236, 50, 4, 211, 92, 192, 17, 134, 144, 168, 106, 126, 235, 101, 133,
-          156, 66, 66, 39, 248, 210, 14, 251, 91, 86, 59, 29, 153, 150, 190>>,
-        <<147, 131, 218, 194, 163, 243, 40, 42, 172, 5, 190, 120, 23, 16, 43, 0, 249,
-          175, 101, 170, 182, 11, 49, 209, 39, 253, 111, 114, 182, 253, 131, 31>>
-      }
+    <<4, 163, 213, 138, 149, 50, 37, 22, 21, 221, 239, 158, 126, 245, 61, 146, 7,
+    15, 86, 26, 224, 169, 46, 191, 199, 39, 172, 189, 146, 10, 111, 160, 51, 7,
+    33, 236, 50, 4, 211, 92, 192, 17, 134, 144, 168, 106, 126, 235, 101, 133,
+    156, 66, 66, 39, 248, 210, 14, 251, 91, 86, 59, 29, 153, 150, 190>>,
+    <<147, 131, 218, 194, 163, 243, 40, 42, 172, 5, 190, 120, 23, 16, 43, 0, 249,
+    175, 101, 170, 182, 11, 49, 209, 39, 253, 111, 114, 182, 253, 131, 31>>
+    }
 
     {account1, account2, account3}
   end
 
   defp get_account_locked_amount() do
     {<<4, 55, 160, 38, 64, 182, 216, 237, 37, 115, 115, 235, 25, 35, 106, 13, 194,
-       87, 156, 61, 156, 235, 207, 151, 183, 35, 38, 247, 66, 253, 39, 197, 43, 49,
-       55, 78, 125, 31, 45, 38, 203, 156, 1, 206, 235, 241, 50, 140, 195, 38, 19,
-       89, 234, 69, 251, 211, 208, 29, 72, 99, 90, 90, 212, 128, 105>>,
+     87, 156, 61, 156, 235, 207, 151, 183, 35, 38, 247, 66, 253, 39, 197, 43, 49,
+     55, 78, 125, 31, 45, 38, 203, 156, 1, 206, 235, 241, 50, 140, 195, 38, 19,
+     89, 234, 69, 251, 211, 208, 29, 72, 99, 90, 90, 212, 128, 105>>,
      <<132, 73, 10, 38, 77, 68, 7, 72, 211, 181, 33, 176, 209, 113, 210, 159, 247,
-       148, 237, 83, 238, 200, 99, 252, 175, 107, 11, 95, 114, 133, 149, 168>>}
+     148, 237, 83, 238, 200, 99, 252, 175, 107, 11, 95, 114, 133, 149, 168>>}
   end
 
   defp create_signed_tx(from_acc, to_acc, value, nonce, fee, lock_time_block \\ 0) do
     {from_acc_pub_key, from_acc_priv_key} = from_acc
     {to_acc_pub_key, _to_acc_priv_key} = to_acc
     {:ok, tx_data} = SpendTx.create(from_acc_pub_key, to_acc_pub_key, value,
-                                   nonce, fee, lock_time_block)
-    {:ok, signature} = Keys.sign(tx_data, from_acc_priv_key)
+      nonce, fee, lock_time_block)
+    {:ok, %{signature: signature}} = SignedTx.sign_tx(tx_data, from_acc_priv_key)
```

Comment:
Here you are extracting the signature from SignedTx just to create the signed_tx again. You should instead:
```
    {:ok, signed_tx} = SignedTx.sign_tx(tx_data, from_acc_priv_key)
    signed_tx
end
```